R/type-vctr.R

```diff
@@ -420,15 +420,27 @@ quantile.vctrs_vctr <- function(x, ..., type = 1, na.rm = FALSE) {
 min.vctrs_vctr <- function(x, ..., na.rm = FALSE) {
   # TODO: implement to do vec_arg_min()
   rank <- xtfrm(x)
-  idx <- if (isTRUE(na.rm)) which.max(rank) else which(rank == min(rank))
+
+  idx <- if (isTRUE(na.rm)) {
```

Comment:
You should move `idx <- ` inside the if clauses (we don't have a principle for this, but should)